modules/arm_plugin/armplg_build.sh

```diff
@@ -149,33 +149,12 @@ cmake -DInferenceEngineDeveloperPackage_DIR=$OPENVINO_HOME/build \
       -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DENABLE_DATA=OFF \
       -DCMAKE_EXE_LINKER_FLAGS=-Wl,-rpath-link,$STAGING_DIR/opencv/lib \
       -DCMAKE_TOOLCHAIN_FILE="$OPENVINO_HOME/cmake/$TOOLCHAIN_DEFS" \
-      -DCMAKE_STAGING_PREFIX=$STAGING_DIR \
+      -DTHREADS_PTHREAD_ARG="-pthread" \
```

Comment:
why do we need it?

Reply:
This flag is used in OpenVINO build procedure ([link](https://github.com/openvinotoolkit/openvino/wiki/BuildingForRaspbianStretchOS#cross-compilation-using-docker), the step 6iii)
I asked Dmitry Kurtaev, he said that the flag is needed for cmake, however the package is built fine without this flag and I tested one OMZ demo - it works fine as well. So, I believe, we may delete it.